integration-tests/utils/integration-utils.js

```diff
@@ -362,18 +362,22 @@ const withScreenshot = _.curry((testName, fn) => async options => {
 
 // Emitted when the page crashes
 const logError = page => {
-  const handle = msg => {
-    console.error('page.error', typeof msg === 'object' ? JSON.stringify(msg) : msg)
-  }
+  // this error will log an object of type [Error: ...] which is not very
+  // informative and looks incorrect at first glance since the contents would be
+  // more useful but in fact this is the best we can do here since the object
```

Comment:
I wouldn't say the object contents are empty. It seems that JSON.stringify doesn't work too well with Error objects.

For example, creating an error in the console. The error has properties, but JSON.stringify does not output them.
```js
const e = new Error('Something went wrong')

e.message
// 'Something went wrong'
e.stack
// 'Error: Something went wrong\n    at <anonymous>:1:11'

JSON.stringify(e)
// '{}'
```